ads/google/doubleclick.js

```diff
@@ -79,6 +79,15 @@ function doubleClickWithGpt(global, data, gladeExperiment) {
     parseInt(data.overrideHeight || data.height, 10),
   ]];
 
+  // If the ad layout is responsive, center the fixed size ad in the container.
+  if (global.context.layout == 'responsive') {
```

Comment:
Good point, centering in an ad frame the same size won't do anything. Do you foresee needing to know the layout for responsive multi-size use case? If not, I can get rid of it from window.context